packages/m2m-gateway/src/services/attributeService.ts

```diff
@@ -70,5 +70,33 @@ export function attributeServiceBuilder(clients: PagoPAInteropBeClients) {
         logger,
       });
     },
+    async getCertifiedAttributes(
+      { limit, offset }: m2mGatewayApi.GetCertifiedAttributesQueryParams,
+      { headers, logger }: WithLogger<M2MGatewayAppContext>
+    ): Promise<m2mGatewayApi.CertifiedAttributes> {
+      logger.info(
+        `Retrieving certified attributes with limit ${limit} and offset ${offset}`
+      );
+
+      const response = await clients.attributeProcessClient.getAttributes({
+        queries: {
+          limit,
+          offset,
+          kinds: [attributeRegistryApi.AttributeKind.Values.CERTIFIED],
+        },
```

Comment:
Let's create and put this part inside a  `toGetCertifiedAttributesApiQueryParams` function in `/api/attributeApiConverter.ts`